### ISSUE TITLE
Remove directional instructions from help text

### ISSUE THREAD

**Issue by nlhkabu** [pypa/warehouse#6249]
WCAG 1.3.3:

> Instructions do not rely upon shape, size, or visual location (e.g., "Click the square icon
to continue" or "Instructions are in the right-hand column".)

To fix:

- Search templates for the words "above", "below".  There are several places where the help text should be rewritten to avoid these terms.

We should also add a section to [our documentation](https://warehouse.readthedocs.io/ui-principles/#write-clearly-with-consistent-style-and-terminology) to ensure new interfaces are not written with location/directional instructions.
